content/loq9/drone-nomad/index.md

```diff
@@ -0,0 +1,127 @@
+---
+date: 2019-10-22T00:00:00+00:00
+title: Nomad
+author: loq9
+tags: [ nomad, orchestration ]
+logo: nomad.svg
+repo: loq9/drone-nomad
+image: plugins/nomad
```

Comment:
This image name can't be true, only images built at the drone-plugins org are published with that name.

Reply:
`loq9/drone-nomad` should be right.

Reply:
You're right, I followed the drone official plugins to do this one. I will change it